pkg/virtctl/vm/vm.go

```diff
@@ -289,6 +320,20 @@ func usage(cmd string) string {
 	return usage
 }
 
+func usageExpand() string {
+	usage := `  #Expand a virtual machine called 'myvm'.
+  {{ProgramName}} expand myvm
+  
+  # Expand a virtual machine from file called myvm.yaml.
+  {{ProgramName}} expand --file myvm.yaml
+
+  # Expand a virtual machine called myvm and display output in json format.
+  {{ProgramName}} expand myvm --output json
```

Comment:
I'm not sure about using a positional arg for the VM here, I'd rather be explicit and have a `--vm` arg or something and mark it as mutually exclusive with `--file` with `MarkFlagsMutuallyExclusive`.

Reply:
Might be another possibility, which also avoids adding additional validation code. But IMO I like both variants.